ssr/progress-bar.js

```diff
@@ -9,6 +9,14 @@ export default class ProgressBar {
     if (includeMemory) {
       this.barLength -= 10;
     }
+    this.bottom = process.stdout.rows;
+    this.quiet = quiet;
+    /**
+     * Draw progress bar at bottom of the terminal
+     * Idea is to reserve a few lines at terminal's bottom
+     * to draw this out, and return the cursor back
+     */
+    this.reservedRows = 5;
```

Comment:
Why 5? Umm not really sure, just an arbitrary number. 2 is definitely too low, since the console.logs from other place could "bleed out" to this area and messes up the drawings

![Annotation 2019-11-27 231248](https://user-images.githubusercontent.com/3090380/69735295-7298a280-116b-11ea-8de6-b068a38247b9.png)

Reply:
That comment is great and should probably be in the code instead of here in the PR on GitHub.